doc/release-notes.md

```diff
@@ -53,6 +53,25 @@ the Linux kernel, macOS 10.8+, and Windows Vista and later. Windows XP is not su
 Bitcoin Core should also work on most other Unix-like systems but is not
 frequently tested on them.
 
+Notes for 0.15.0
+================
+
+Current SegWit Support
+----------------------
+
+Version 0.15.0 supports adding a segregated witness address via the `addwitnessaddress` RPC, but
+please note that this is a testing/expert RPC, which does not guarantee recovery from backup. Only use
+this RPC if you know what you are doing. More complete support for segregated witness, including
+wallet support, is coming in a next version.
+
+Rescanning with encrypted wallets
+---------------------------------
+
+When using an encrypted HD wallet, the keypool cannot be topped up without unlocking the wallet. This means that
+currently, in order to recover from a backup of an encrypted HD wallet, the user must unlock the wallet with a
+really long time out and manually trigger a rescan with `-rescan`, otherwise they risk missing some keys when
```

Comment:
Rescan rpc (or command in the gui debug console).  ... if you restart with -rescan you'll be relocked!

Reply:
Oops, fixed, thanks